commit 4664c6ae706543c340199047f759bf7b0b6b71d1
Author: popcornmix <popcornmix@gmail.com>
Date:   Thu Apr 7 12:44:24 2016 +0100

    config: Enable CONFIG_IPV6_ROUTER_PREF for networks with multiple routers

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 299d7be..7bfc073 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -101,6 +101,7 @@ CONFIG_INET_XFRM_MODE_TUNNEL=m
 CONFIG_INET_XFRM_MODE_BEET=m
 CONFIG_INET_LRO=m
 CONFIG_INET_DIAG=m
+CONFIG_IPV6_ROUTER_PREF=y
 CONFIG_INET6_AH=m
 CONFIG_INET6_ESP=m
 CONFIG_INET6_IPCOMP=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index acda16b..59026b3 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -95,6 +95,7 @@ CONFIG_INET_XFRM_MODE_TUNNEL=m
 CONFIG_INET_XFRM_MODE_BEET=m
 CONFIG_INET_LRO=m
 CONFIG_INET_DIAG=m
+CONFIG_IPV6_ROUTER_PREF=y
 CONFIG_INET6_AH=m
 CONFIG_INET6_ESP=m
 CONFIG_INET6_IPCOMP=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1236_4664c6ae706543c340199047f759bf7b0b6b71d1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1236_4664c6ae706543c340199047f759bf7b0b6b71d1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
